homeassistant/components/rki_covid/__init__.py

```diff
@@ -0,0 +1,172 @@
+"""RKI Covid numbers integration."""
+import asyncio
+from datetime import timedelta
+import logging
+
+import aiohttp
+import async_timeout
+from rki_covid_parser.parser import RkiCovidParser
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import aiohttp_client, update_coordinator
+
+from .const import DOMAIN
+from .data import DistrictData, StateData
+
+_LOGGER = logging.getLogger(__name__)
+PLATFORMS = ["sensor"]
+
+
+async def async_setup(hass: HomeAssistant, config: dict) -> bool:
+    """Set up the component into HomeAssistant."""
+    _LOGGER.debug("setup component.")
+    parser = RkiCovidParser(aiohttp_client.async_get_clientsession(hass))
+
+    # Make sure coordinator is initialized.
+    await get_coordinator(hass, parser)
+
+    # Return boolean to indicate that initialization was successful.
+    return True
```

Comment:
Can be removed since loading from config entry

Reply:
The whole block or just the return?

Reply:
The whole block can go as we it should only be used from a config entry

Reply:
done